### PR TITLE
[DOCFIX] Add additional requirement for fuse metrics

### PR DESCRIPTION
**Pull request by ssz1997**
To check standalone Fuse metrics, `alluxio.fuse.web.enabled` needs to be set to `true` in `alluxio-site.properties`. Add this to the Posix-api doc.

### CODE REVIEW

**Comment by ssz1997** [comment]
@LuQQiu PTAL. Thanks!

File: docs/en/api/POSIX-API.md [hunk 1]
```diff
@@ -515,9 +515,9 @@ $ ./bin/alluxio logLevel --logName=alluxio.fuse --target=workers --level=DEBUG
 For more information about logging, please check out [this page]({{ '/en/operation/Basic-Logging.html' | relativize_url }}).
 
 ### Fuse metrics
-
-Check out the [Fuse metrics doc]({{ '/en/reference/Metrics-List.html' | relativize_url }}#fuse-metrics) for how to get Fuse metrics
-and what each metric uses for.
+To monitor Fuse-related metrics for standalone Fuse process, setting `alluxio.fuse.web.enabled` to `true` in `${ALLUXIO_HOME}/conf/alluxio-site.properties`.
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:518]: setting..... before launching the standalone fuse process

**Review by LuQQiu — APPROVED** [review]
Left a minor comment, thanks

File: docs/en/api/POSIX-API.md (hunk 1 above)
Comment (ssz1997) [reply docs/en/api/POSIX-API.md:518]: Probably before launching Alluxio?

**Review by LuQQiu — APPROVED** [review]
LGTM, waiting for the build to pass

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
@Xenorith cherry-pick?

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**